docs/resources/new_cluster.plantuml

```diff
@@ -0,0 +1,24 @@
+@startuml
+title Add new cluster
+
+actor "OSIO user" as user
+box "OSD Cluster" #LightBlue
+	participant "cluster API" as cluster
+end box
+box "OSIO"
+	participant "Cluster Service" as cs
+	participant "Auth Service" as auth
+	participant "UI\nuser settings page" as usersettings
+end box
+
+user -> usersettings : Add cluster API URL
+group link OSIO - OSD accounts
+	usersettings -> auth : GET /token/link?for={clusterAPI}
+	auth --> cs : get cluster
+	auth <-- cs
+	auth --> cluster
+	cluster --> user : autenticate
+	cluster --> auth : {user_token}
+	auth -> cs : Create new identity_cluster\nPOST /clusters/identity
```

Comment:
It will help if we add payload details for this call.  As I got, it would be cluster, user_token but not quite sure.